README.md

```diff
@@ -23,11 +31,12 @@ Install the following dependencies if you don't have them installed yet
 
  - [Node.js](https://nodejs.org/en/download/)
  - [Git for Windows](https://git-for-windows.github.io/)
- 
+
 #### Set up node
- 
+
  - Open cmd window and navigate into `ServicePulse\src\ServicePulse.Host` path
- - run the following command `npm run setup`. This script will install all node and bower dependencies and update index.html to include them if needed.
+ - run the following command `npm install` to ensure all required dependencies are available
```

Comment:
This line should not be neccessary as `npm run setup` include `npm install`

Reply:
True, however on a clean machine it was constantly failing with the following error:

```
λ  [master ≡]npm run setup

> service-pulse@1.0.0 setup C:\dev\particular\ServicePulse\src\ServicePulse.Host
> npm install && node ./node_modules/webpack/bin/webpack.js --config app/modules/modules.webpackconfig.js

npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm ERR! code 128
npm ERR! Command failed: C:\Program Files\Git\cmd\git.EXE submodule update -q --init --recursive
npm ERR! fatal: 'submodule' appears to be a git command, but we were not
npm ERR! able to execute it. Maybe git-submodule is broken?
npm ERR!

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mauro\AppData\Roaming\npm-cache\_logs\2019-03-01T09_37_46_635Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! service-pulse@1.0.0 setup: `npm install && node ./node_modules/webpack/bin/webpack.js --config app/modules/modules.webpackconfig.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the service-pulse@1.0.0 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mauro\AppData\Roaming\npm-cache\_logs\2019-03-01T09_37_46_712Z-debug.log
```

running `npm install` in isolation first was the only way to get it working.